templates/company-board/README.md

```diff
@@ -0,0 +1,51 @@
+# Aragon Company Board template
+
+## Usage
+
+```
+company.prepareInstance(id)
+company.setupInstance(boardHolders, shareHolders, shareStakes)
+```
+
+- `id`: Identifier to be registered in AragonID
+- `boardMembers`: Array of board member addresses (1 token will be minted for each board member)
+- `shareHolders`: Array of share holder addresses
+- `shareStakes`: Array of token stakes for share holders (token has 18 decimals, multiply token amount `* 10^18`)
+
+## Deploying templates
+
+After deploying ENS, APM and AragonID, just run:
+
+```
+npm run deploy:rinkeby
+```
+
+The network details will be automatically selected by the `arapp.json`'s environments.
+
+## Permissions
+
+| App                 | Permission            | Grantee             | Manager       |
+|---------------------|-----------------------|---------------------|---------------|
+| Kernel              | APP_MANAGER           | Board Voting        | Share Voting  |
+| ACL                 | CREATE_PERMISSIONS    | Board Voting        | Share Voting  |
+| Board Voting        | CREATE_VOTES          | Board Token Manager | Share Voting  |
+| Board Voting        | MODIFY_QUORUM         | Share Voting        | Share Voting  |
+| Board Voting        | MODIFY_SUPPORT        | Share Voting        | Share Voting  |
+| Share Voting        | CREATE_VOTES          | Board Token Manager | Share Voting  |
```

Comment:
So Share holders have all the power (like being Permission manager of everything, or being able to mint and burn Board tokens) but through their voting app, which votes can only be created by board members. So if board members collude they could keep all the effective power, right?

Reply:
If 100 percent of board members collude then yes they can prevent shareholders from doing anything, but if just one board member is honest then it should not be an issue. We could make it more permissive and allow any shareholder to create a vote... or use the tollgate app to allow for an "emergency" shareholder vote... but I think in most cases the assumption that there is atleast one honest board member is reasonable and by restricting vote creation to them it is easier for the organization to implement off-chain processes for how votes should or should not be created, how shareholders should be notified, etc. (the reason being that a board member who does not follow the process to create a vote can be removed). 

Anyways that was the rationale, hopefully it makes sense. :)

Reply:
Shareholders are still the role managers and have some access to functionality, so they could technically grant themselves various powers (minting a new board member, granting themselves access to create votes, etc.) if they needed to.

Reply:
> ~Shareholders are still the role managers and have some access to functionality, so they could technically grant themselves various powers (minting a new board member, granting themselves access to create votes, etc.) if they needed to.~
> 
> Nevermind, I see shareholder votes can only be created by board token holders.

Yes, that was actually my concern.

Although I guess that the assumption that at least one board member is honest, I don't fully get the problem of assigning `CREATE_VOTE_ROLE` to shareholders too.

Reply:
Mostly spam, probably. Typically shareholders do not get to create votes themselves, unless they are a fairly large holder (at which point the board is most likely theirs anyway).

You'd typically have another set of people, the management, that would also have control over creating votes / budgets, and that could be introduced via another token (if not an entire sub organization with its own finance app and etc.).